packages/perseus/src/widgets/interactive-graphs/graphs/utils.ts

```diff
@@ -237,3 +239,103 @@ export function getQuadraticXIntercepts(
 
     return [x1, x2];
 }
+
+export function getAngleFromPoints(points: Coord[], i: number) {
+    if (i < 0 || i >= points.length || !Number.isInteger(i)) {
+        return null;
+    }
+
+    if (points.length < 3) {
+        return null;
+    }
+
+    const point = points.at(i);
+    const pt1 = points.at(i - 1);
+    const pt2 = points[(i + 1) % points.length];
+    if (!point || !pt1 || !pt2) {
+        return null;
+    }
+
+    const a = vec.dist(point, pt1);
+    const b = vec.dist(point, pt2);
+    const c = vec.dist(pt1, pt2);
+
+    // Law of cosines
+    const angle = Math.acos((a ** 2 + b ** 2 - c ** 2) / (2 * a * b));
+
+    return angle;
+}
+
+export function radianToDegree(radians: number) {
```

Comment:
I could have sworn we already had a function for this, possibly in kmath?

Reply:
Noting here for posterity: kmath had `convertDegreesToRadians` but not the other way around. I added `convertRadiansToDegrees` in another PR and pulled it in here.